BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.kt

```diff
@@ -216,13 +216,20 @@ open class BraintreeClient @VisibleForTesting internal constructor(
     fun sendAnalyticsEvent(
```

Comment:
Looks like this method signature is growing on a weekly basis.
Would it make sense to extract all the payload into an object of its own so we can keep the method signature same?

Reply:
+1 There is an open ticket to investigate moving to a SQL DB since it keeps changing - but maybe a quick win to add an `AnalyticsParams` object or something like that so we don't have to update so many classes/tests every time

Reply:
Of course, happy to work on this and create an object with the parameters, I would just like to work on it in another PR, since several classes/unit tests will be modified

Reply:
Yes agreed - lets just make a ticket for that work and can address it at lower priority